app.rb

```diff
@@ -122,7 +128,12 @@ def config
 
     private
 
-    def authorization_url(ial:, aal: nil)
+    def authorization_url(session:, ial:, aal: nil)
+      if ial == 'step-up'
+        ial = '1'
+        session[:step_up_enabled] = 'true'
+        session[:step_up_aal] = aal
```

Comment:
The session side-effect here is a bit unexpected to me, for a method would would seem to purely generate a URL. Also, do we need to assign to the session for the `get '/'` handling, or is it enough to do this inline for the `get '/auth/request'` handler, since that's where the form is submitting?

Reply:
Ohh, I see we also use this `ial_url` for the "Create an account" link. Hmm... 🤔

Reply:
Good call - moved the step-up prep to a separate method in e65cbd6